lib/WasmReader/WasmBinaryReader.cpp

```diff
@@ -235,19 +273,24 @@ WasmBinaryReader::ReadFunctionBodies(FunctionBodyCallback callback, void* callba
         {
             UINT32 count = LEB128(len);
             m_funcState.count += len;
-            UINT8 type = ReadConst<UINT8>();
-            m_funcState.count++;
-            if (type >= Wasm::WasmTypes::Limit || type == Wasm::WasmTypes::Void)
+            Wasm::WasmTypes::WasmType type = ReadWasmType(len);
```

Comment:
I suppose we don't at this time, this is an effort to minimize possible errors if it does change, plus it looks like the way we use LEB128.